.github/ci-windows-latest.yaml

```diff
@@ -0,0 +1,92 @@
+
+# Just a work in progress version, not fully implemented. For conceptual purpose only.
+# Need to add more steps and tested to make it functional fully. But this can be achieved.  
+
+name: ci-windows-latest
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'master'
+
+jobs:
+  build-docker-latest:
+    runs-on: windows-2019
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: ./rabbitmq/windows/latest
+          push: true
+          tags: micdenny/rabbitmq-windows:latest
+
+  build-docker-38:
```

Comment:
is there something like [matrix strategy of azure devops pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#multi-job-configuration)? do avoid having to copy and paste for every version? or something like [each keyword](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#each-keyword)?

Reply:
we can also do it later